frame/contracts/src/exec.rs

```diff
@@ -740,7 +740,7 @@ where
 	}
 
 	fn random(&self, subject: &[u8]) -> SeedOf<T> {
```

Comment:
Do we want to change the contracts API to also expose the block number when randomness was known?

Reply:
I think we pretty much need to unless we only expose `pallet_babe::CurrentBlockRandomness<Runtime>;`

Reply:
cc @athei

Reply:
This would need deprecating `seal_random` and create a v2 which returns both. But I see no reason why not to do that.